package.xml

```diff
@@ -1,20 +1,29 @@
 <?xml version="1.0"?>
-<package format="2">
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>pacmod2</name>
-  <version>3.0.0</version>
-  <description>AutonomouStuff PACMod driver package</description>
-  <license>MIT</license>
-
+  <version>2002.1.0</version>
```

Comment:
Any reason we aren't calling this version 2003.0.0?

Reply:
changed it